pages/ArticlesByTopic.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from "react";
```

Comment:
Ideally you shouldn't need a new component to render articles by topic, you should be using the same component to render all articles regardless of a query or not.